other/materials_designer/create_monolayer.ipynb

```diff
@@ -0,0 +1,271 @@
+{
```

Comment:
<p>let's call the variable <code>monolayer</code> instead of slab? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/161/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>